storage/src/main/java/io/delta/storage/commit/CoordinatedCommitsUtils.java

```diff
@@ -120,10 +120,17 @@ public static Path commitDirPath(Path logPath) {
         return new Path(logPath, COMMIT_SUBDIR);
     }
 
-    public static String getCoordinator(AbstractMetadata metadata) {
+    /**
+     * Retrieves the coordinator name from the provided abstract metadata.
+     * If no coordinator is set, an empty string is returned.
```

Comment:
Given that we've changed the signature to an `Optional` now, should this say "an empty optional" instead? Same in line 134 below?